src/courtroom.cpp

```diff
@@ -4405,6 +4414,32 @@ void Courtroom::on_pos_dropdown_changed(QString p_text)
   set_side(p_text);
 }
 
+void Courtroom::on_pos_dropdown_context_menu_requested(const QPoint &pos)
+{
+  QMenu *menu = ui_iniswap_dropdown->lineEdit()->createStandardContextMenu();
+
+  menu->setAttribute(Qt::WA_DeleteOnClose);
+  menu->addSeparator();
+
+  menu->addAction(QString("Open background " + current_background), this,
+                  [=] {
+    QString p_path = ao_app->get_real_path(VPath("background/" + current_background + "/"));
+    if (!dir_exists(p_path)) {
+        return;
+    }
+  }
+  );
+  menu->addAction(QString("Open background folder"), this,
+                  [=] {
+    QString p_path = ao_app->get_real_path(VPath("background/"));
+    if (!dir_exists(p_path)) {
+        return;
+    }
+  }
+  );
+  menu->popup(ui_iniswap_dropdown->mapToGlobal(pos));
+}
```

Comment:
These actions do nothing when clicked on. Also has the same issue as the theme folder.